rotkehlchen/tests/unit/globaldb/test_globaldb_consistency.py

```diff
@@ -371,3 +376,87 @@ def test_oracle_ids_in_asset_collections(globaldb: 'GlobalDBHandler'):
+        UpdateType.LOCATION_ASSET_MAPPINGS,
+        UpdateType.LOCATION_UNSUPPORTED_ASSETS,
+    ])
+
+    with (
+        globaldb.conn.read_ctx() as old_db_cursor,
+        globaldb.packaged_db_conn().read_ctx() as packaged_db_cursor,
+    ):
+        (
+            (updated_location_asset_mappings, updated_location_unsupported_assets),
+            (packaged_location_asset_mappings, packaged_location_unsupported_assets),
+        ) = (
+            (
+                {
+                    (location, symbol): identifier
+                    for location, symbol, identifier in cursor.execute(
+                        'SELECT location, exchange_symbol, local_id FROM location_asset_mappings',
+                    )
+                }, set(cursor.execute(
+                    'SELECT location, exchange_symbol FROM location_unsupported_assets',
+                ).fetchall()),
+            )
+            for cursor in (old_db_cursor, packaged_db_cursor)
+        )
+
+    # find all the location_asset_mappings that are present in remote updates but not in packaged db  # noqa: E501
+    missing_in_packaged_db = [
+        f"INSERT INTO location_asset_mappings(local_id, location, exchange_symbol) VALUES('{identifier}', '{location}', '{symbol}');"  # noqa: E501
+        for (location, symbol), identifier in updated_location_asset_mappings.items()
+        if (location, symbol) not in packaged_location_asset_mappings
+    ]
+
+    # find all the location_asset_mappings that are present in packaged db but not in remote updates  # noqa: E501
+    missing_in_remote_updates = [
+        {
+            'asset': identifier,
+            'location': None if location is None else Location.deserialize_from_db(location).serialize(),  # noqa: E501
+            'location_symbol': symbol,
+        }
+        for (location, symbol), identifier in packaged_location_asset_mappings.items()
+        if (location, symbol) not in updated_location_asset_mappings
+    ]
+
+    # find all the location_unsupported_assets that are present in remote updates but not in packaged db  # noqa: E501
+    missing_in_packaged_db.extend([
+        f"INSERT INTO location_unsupported_assets(location, exchange_symbol) VALUES('{location}', '{symbol}');"  # noqa: E501
+        for location, symbol in updated_location_unsupported_assets
+        if (location, symbol) not in packaged_location_unsupported_assets
+    ])
+
+    # find all the location_unsupported_assets that are present in packaged db but not in remote updates  # noqa: E501
+    missing_unsupported_assets = defaultdict(list)
+    for location, symbol in packaged_location_unsupported_assets:
+        if (location, symbol) not in updated_location_unsupported_assets:
+            missing_unsupported_assets[Location.deserialize_from_db(location).serialize()].append(symbol)
+
+    if len(missing_unsupported_assets) > 0:
+        missing_in_remote_updates.append(missing_unsupported_assets)
+
+    if len(missing_in_packaged_db) > 0:
```

Comment:
Why not fail here?

Reply:
This is because, generally we keep adding the remote updates whenever we want to add an asset, without touching the packaged db. Then at the end just before releasing, we add all of them together in the packagedb at once (final packaging). So, we don't fail it here otherwise it will always fail and we will have to do twice PRs whenever we add an update (one extra PR each to add in the packaged db).